core/vm/instructions.go

```diff
@@ -516,8 +516,10 @@ func opGasprice(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([
 	return nil, nil
 }
 
+const Eip2935BlockHashHistorySize = 8096
```

Comment:
it's 8192 and I suggest you add it to `params/protocol_params.go` or `params/verkle_params.go`